app/views/paye/paye_government_spending.scala.html

```diff
@@ -44,7 +44,7 @@ <h1 class="heading-xlarge">
 	<dl class="form-group" id="bar-chart">
 	@for(rowModel <- viewModel.orderedSpendRows){
 		<dt class="percentage" id="@rowModel.category">@Messages("paye.ats.treasury_spending.table." + rowModel.category) (@rowModel.spendData.percentage%)</dt>
-		<dd class="value"><span class="bar-chart" aria-hidden="true" style="width: @{rowModel.spendData.percentage}%;"></span> <strong>&pound;@rowModel.spendData.amount.toHalfRoundedUpAmount</strong></dd>
+		<dd class="value"><span class="bar-chart" aria-hidden="true" style="width: @{rowModel.spendData.percentage*2}%;"></span> <strong>&pound;@rowModel.spendData.amount.toHalfRoundedUpAmount</strong></dd>
```

Comment:
This will look off if any of the percentages are over 51 (in the future) as it'll make the bar chart overflow

Is there a way to normalise the data to fit into the width better?

Reply:
Hi Steven, I agree. I was asked to make the bar charts wider. I'm hoping that in the future, we do away with this poor implementation of bar charts, which will fix this and many other issues. We could use a smaller multiplier such as x1.5. But I'd prefer to focus on the future of when we do away with this and improve from the ground up. But that annoyingly (from my point of view) will have to wait until after MVP. So I'm stuck between a rock and a hard place. If this is ok for now, I'd like to leave and push to quicker implement a better thing.

Reply:
@russellthorn can you leave a comment with the ticket number on for when this will be fixed here for future ref

Reply:
@oc-ste Completely agree with your point.
Here is the ticket number ATS-259